src/components/templates/Tables/AddedUsersTable/AddedUsersTable.tsx

```diff
@@ -0,0 +1,116 @@
+import { FC, useState, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import DataTable, {
+  TableSizeTypes,
+} from 'components/organisms/DataTable/DataTable'
+import { map, join } from 'lodash'
+import {
+  createColumnHelper,
+  PaginationState,
+  ColumnDef,
+} from '@tanstack/react-table'
+import Button, {
+  AppearanceTypes,
+  SizeTypes,
+  IconPositioningTypes,
+} from 'components/molecules/Button/Button'
+
+import { ReactComponent as ArrowRight } from 'assets/icons/arrow_right.svg'
+import users from 'components/templates/Tables/users.json'
+
+type Person = {
+  id: string
+  name: string
+  department: string | null
+  roles: string[]
+  status: string
+  subRows?: Person[]
+}
+
+const columnHelper = createColumnHelper<Person>()
+
+const AddedUsersTable: FC = () => {
+  const { t } = useTranslation()
+  const [pagination, setPagination] = useState<PaginationState>({
+    pageIndex: 0,
+    pageSize: 10,
+  })
+  const tableData = users?.data
+
+  const usersData = useMemo(() => {
+    return (
+      map(tableData, (data) => {
+        const arrayOfRoles = map(data.roles, ({ name }) => name)
```

Comment:
**[non-blocker]**

With lodash map, if you just want 1 key from the array you can use the shorthand: map(data.roles, 'name')